tests/frontend/test_versions.py

```diff
@@ -0,0 +1,139 @@
+import time
+from datetime import datetime
+
+import pytest
+import requests
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from pages.desktop.frontend.versions import Versions
+
+ADDON_VERSION_PAGE = '/addon/devhub-listed-ext-07-30/versions/'
+
+
+@pytest.mark.nondestructive
+def test_addon_name_in_header(selenium, base_url):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+    addon_name = page.versions_page_header.text.split()[0]
+    addon_detail_page = page.rating_card.click_addon_title()
+    assert addon_name in addon_detail_page.name
+
+
+@pytest.mark.nondestructive
+def test_versions_counter(selenium, base_url):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+    text = page.versions_page_header.text
+    text = text.split('-')[-1][1:]
+    text = text.split('versions')[0][:-1]
+    assert int(text) == len(page.versions_list)
+
+
+@pytest.mark.nondestructive
+def test_notice_message(selenium, base_url, variables):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+    assert page.notice_message.is_displayed()
+    assert variables['version_page_notice_message'] in page.notice_message.text
+
+
+@pytest.mark.nondestructive
+def test_ratings_card(selenium, base_url, variables):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+    # verify if ratings card is present
+    assert page.rating_card
+
+
+@pytest.mark.nondestructive
+def test_license_link(selenium, base_url):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+
+    for i in range(len(page.versions_list)):
+        if page.versions_list[i].license_link is not False:  # if link exists
+
+            if (
+                page.versions_list[i].license_link.text == 'Custom License'
+            ):  # if link exists and is 'Custom License'
+                page.versions_list[i].license_link.click()
+                page.wait.until(
+                    EC.visibility_of_element_located(
+                        (By.CLASS_NAME, 'AddonInfo-info-html')
+                    )
+                )
+
+            else:  # if link exists and is not 'Custom License'
+                expected_link = page.versions_list[i].license_link.get_attribute('href')
+                if 'http' or 'https' in expected_link:
+                    # full url example-> https://example.com/something/something
+                    # turn into-> example.com/something/something
+                    expected_link = expected_link.split('//')[1]
+                    # turn into-> example.com
+                expected_link = expected_link.split('/')[0]
```

Comment:
It cannot be done without splitting because there could be differences inside the links.
For example:
 'http://www.mozilla.org/MPL/2.0/ '                - link in the html code
 'https://www.mozilla.org/en-US/MPL/2.0/'   - link from the opened page